src/components/common/Members/MembersTitle.tsx

```diff
@@ -36,6 +36,32 @@ const MSG = defineMessages({
   },
 });
 
+const handleSearchIconFocus = (searchInput: RefObject<HTMLInputElement>) => {
+  searchInput?.current?.focus();
+};
+
+const handleSearchIconEnter = (searchInput: RefObject<HTMLInputElement>) => {
+  if (searchInput.current !== null) {
+    const input = searchInput.current;
+    input.placeholder = formatText(MSG.searchPlaceholder) ?? '';
+  }
+};
+
+const handleSearchIconLeave = (searchInput: RefObject<HTMLInputElement>) => {
+  if (searchInput.current !== null) {
+    const input = searchInput.current;
+    input.placeholder = '';
+  }
+};
+
+const handleMouseEnter = (e: MouseEvent<HTMLInputElement>) => {
+  e.currentTarget.placeholder = formatText(MSG.searchPlaceholder) as string;
+};
+
+const handleMouseLeave = (e: MouseEvent<HTMLInputElement>) => {
+  e.currentTarget.placeholder = '';
+};
```

Comment:
Could that be possibly all handled in CSS with `:hover` and `::placeholder`?

Reply:
I agree with this. I'd rather have effects like these handled by CSS

_(If they don't work outright, feel free to look into adding post-css addons that handle that)_

Reply:
@jakubcolony I found a way to handle the hover and placeholder cases but I wasn't able to trigger a change of focus with only css. Do you think there is a way to do that as well?

Reply:
Let me make sure I understand what you're trying to achieve, since the naming seems a bit confusing.

You're trying to focus the searchbox on the search icon click (not focus, right?). In that case, you should be able to place both the input and the icon in the same wrapper with position relative, then position the icon so that it is on top of the input, and set its `pointer-events` property to `none`. So all the clicks on the icon will pass through and cause the input to focus.

If that doesn't work or doesn't make sense, feel free to ping me and I'll try to help

Reply:
Here's a demo: https://jsfiddle.net/0cvfkqtr/

Note you can click on the red box (representing the icon) and the input will focus

Reply:
@jakubcolony updated with what you suggested for the search input focus